foruma-frontend/src/stories/Button.stories.js

```diff
@@ -0,0 +1,45 @@
+import MyButton from './Button.vue';
+
+export default {
+  title: 'Example/Button',
+  component: MyButton,
+  argTypes: {
+    backgroundColor: { control: 'color' },
```

Comment:
Codacy has a fix for the issue: [Replace `'color'` with `"color"`](https://app.codacy.com/gh/agabani/foruma/pullRequest?prid=7388752)
```suggestion
    backgroundColor: { control: "color" },
```